mlir/test/fusion/pr-e2e/tosa-to-rock-exp.e2e.mlir

```diff
@@ -2,8 +2,7 @@
 // ALLOW_RETRIES: 2
 
 module {
-// CLONE: RMS = {{.*}}e-09
-// CLONE: [1 0 0]
+// CLONE: [1 1 1]
```

Comment:
I am not sure what `CLONE` checks, but i've instead seen `CHECK: [1 1 1]` in other `pr-e2e` tests.

Reply:
Above in the `RUN:` directive, we pass `--check-prefix=CLONE` to FileCheck. It's explained here: https://llvm.org/docs/CommandGuide/FileCheck.html#the-filecheck-check-prefix-option
It doesn't seem to be useful in this case however.